libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -353,13 +383,22 @@ export class ComboboxComponent implements ControlValueAccessor, OnInit, OnChange
         if (this.popoverComponent) {
             this.popoverComponent.updatePopover();
         }
+        if (this.displayedValues && this.displayedValues.length === 0 && this.inputGroup.inputGroupButtonElement) {
+            this.inputGroup.inputGroupButtonElement.elementRef().nativeElement.classList.remove('is-expanded');
+        } else if (this.inputGroup.inputGroupButtonElement) {
+            this.inputGroup.inputGroupButtonElement.elementRef().nativeElement.classList.add('is-expanded');
+        }
```

Comment:
If you need to get reference to button, or bind something, I encourage to use `fd-input-group-addon` - [example](https://fundamental-ngx.netlify.app/#/core/inputGroup#complex). You would be able to put `<button [ngClass]={'is-expanded": isExpanded} [glyph]="..."></button>`